src/libraries/System.Private.Xml.Linq/src/System/Xml/Linq/XAttribute.cs

```diff
@@ -131,15 +133,16 @@ public override XmlNodeType NodeType
         /// If this attribute does not have a parent, or if there is no previous attribute,
         /// then this property returns null.
         /// </remarks>
-        public XAttribute PreviousAttribute
+        public XAttribute? PreviousAttribute
         {
             get
             {
                 if (parent == null) return null;
-                XAttribute a = ((XElement)parent).lastAttr;
+                XAttribute a = ((XElement)parent).lastAttr!;
                 while (a.next != this)
                 {
-                    a = a.next;
+                    Debugger.Launch();
```

Comment:
🙈

Reply:
I think that the fact that this didn't make any test to hang shows that there is some lack of coverage.